src/experimental/tools/VideoThumbnailLoader/prepare_source_buffer.ts

```diff
@@ -66,6 +69,29 @@ export default function prepareSourceBuffer(
         resetMediaElement(videoElement, objectURL);
       });
     }
+    if (isManagedMediaSource) {
+      const disableRemotePlaybackPreviousValue = videoElement.disableRemotePlayback;
+      cleanUpSignal.register(() => {
+        /**
+         * Restore the disableRemotePlayback attribute to the previous value
+         * in order to leave the <video> element in the same state has it was set
+         * by the application before calling the RxPlayer.
+         */
+        if (
+          "disableRemotePlayback" in videoElement &&
+          disableRemotePlaybackPreviousValue !== undefined
```

Comment:
Why having that condition?

Reply:
- first condition `"disableRemotePlayback" in videoElement` because on browsers that does not have `disableRemotePlayback` I don't see the point of setting it.
- 2nd condition `disableRemotePlaybackPreviousValue !== undefined` is because typescript don't allow to possibly  assign `videoElement.disableRemotePlayback`  to `undefined`. Because it's typed as `disableRemote?: boolean`

Reply:
It was for the second condition.

So it creates a weird situation where we would reset the value UNLESS it was previously set to `undefined`...

Maybe calling something like `removeAttribute` in that case is more logical / simpler to understand?

Reply:
> UNLESS it was previously set to undefined

The property is a boolean so it should never be set to `undefined` 

I can use removeAttribute but this will not reset it to true if that was already true before using the RxPlayer